lua/lspconfig/dotls.lua

```diff
@@ -0,0 +1,35 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = "dotls"
+local bin_name = "dot-language-server"
+
+local root_files = {
+  '.git',
+}
+
+configs[server_name] = {
+  default_config = {
+    cmd = {bin_name, "--stdio"};
+    filetypes = {"dot"};
+    root_dir = function(filename)
+      return util.root_pattern(unpack(root_files))(filename) or
+             util.path.dirname(filename)
+    end;
+  };
+  docs = {
+    description = [[
+https://github.com/nikeee/dot-language-server
+
+`dot-language-server` can be installed via `npm`:
+```sh
+npm install -g dot-language-server
+```
+    ]];
+    default_config = {
+      root_dir = "vim's starting directory";
```

Comment:
This can be either removed or updated :)

Reply:
Killed it. :)

Reply:
FYI the only reason this can be removed is I recently added a commit that automatically captures the defined root dir function in our docgen

Reply:
That's a nice feature. Good to know. Thanks.